### ISSUE TITLE
jrbuddas list of things to work on when his computer is unbroken

### ISSUE THREAD

**Issue by jrbudda** [jrbudda/Vivecraft_112#33]
- ~glowing' effect breaking rendering~
 - Interactable GUI's intermittently get stuck to the hand.

**Comment by jrbudda** [comment]
if someone has #2 happen to them let me know here.

**Comment by crazyirishwrencher** [comment]
I have the gui thing happening to me. Hitting F1 to hide the gui and again to display it fixes it until the next time I open a gui.

**Comment by jrbudda** [comment]
do ALL gui's do it every time you open one?

**Comment by crazyirishwrencher** [comment]
So after I posted this I tried the download link in the other issue (alpha 7). I can't duplicate the issue in this build. Just to make sure I wasnt crazy I reverted to prev (Alpha 3) and the issue came back, but only if I altered the gui binding (Head, Wrist, Arm). Whatever Gui I was in before exiting back to game remained stuck to my view. Anyway, seems fixed in Alpha 7 so I don't know if any of that is of any use to you.
